ambry-utils/src/test/java/com.github.ambry.utils/HelixControllerManager.java

```diff
@@ -0,0 +1,80 @@
+/**
+ * Copyright 2019 LinkedIn Corp. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ */
+package com.github.ambry.utils;
+
+import java.util.concurrent.CountDownLatch;
+import org.apache.helix.InstanceType;
+import org.apache.helix.manager.zk.ZKHelixManager;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+
+public class HelixControllerManager extends ZKHelixManager implements Runnable {
```

Comment:
Is this a copied class?  The style looks different.

Reply:
This is copied from Helix code. Will covert it to our style.

Reply:
Followed the Redistribution rule in http://www.apache.org/licenses/LICENSE-2.0.html

Reply:
looks better